### PR TITLE
Add Ada Utility Library AWS support for 2.6.0

### PR DESCRIPTION
**Pull request by stcarrez**
This is an old version of Ada Utility Library but the AWS part was not submitted due to problems with AWS. Hopefully, this should work with AWS 23.0.

### CODE REVIEW

**Comment by stcarrez** [comment]
Ah.... this AWS 23.0 is still failing in automated build but it works for me.  This is becoming a nightmare....

**Comment by stcarrez** [comment]
alr 2.1-dev seems broken. It fails to build a correct path to find the `alire.toml` file. Builds with alr 2.0 are passing.

**Comment by mosteo** [comment]
Curious that only Ubuntu builds are failing. I should be able to reproduce this locally.

File: index/ut/utilada_aws/utilada_aws-2.6.0.toml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+description = "Utility Library REST support on top of AWS"
+long-description = "\n[![Build Status](https://img.shields.io/jenkins/s/https/jenkins.vacs.fr/Ada-Util.svg)](https://jenkins.vacs.fr/job/Ada-Util/)\n[![Test Status](https://img.shields.io/jenkins/t/https/jenkins.vacs.fr/Ada-Util.svg)](https://jenkins.vacs.fr/job/Ada-Util/)\n[![codecov](https://codecov.io/gh/stcarrez/ada-util/branch/master/graph/badge.svg)](https://codecov.io/gh/stcarrez/ada-util)\n\n\nThis small library provides an HTTP backend on top of AWS.\nIt is can be used by the `Util.Http` package.\n\nAn alternate HTTP backend is provided by CURL with utilada_curl.\n\n"
+name = "utilada_aws"
+version = "2.6.0"
+authors = ["Stephane.Carrez@gmail.com"]
+licenses = "Apache-2.0"
+maintainers = ["Stephane.Carrez@gmail.com"]
+maintainers-logins = ["stcarrez"]
+project-files = ["utilada_aws.gpr"]
```
Comment (mosteo) [inline index/ut/utilada_aws/utilada_aws-2.6.0.toml:9]: The problem was in this line, that was given as relative to the repo root. I fixed it to be relative to the crate root inside the repo, for future reference.

**Review by mosteo — APPROVED** [review]

**Pull request merged**